scripts/B-FHTL_exp_scripts/Graph-DT/fedbn_gnn_minibatch_on_multi_task.yaml

```diff
@@ -0,0 +1,35 @@
+use_gpu: True
+device: 0
+early_stop:
+  patience: 0
+  improve_indicator_mode: mean
+federate:
+  mode: 'standalone'
+  make_global_eval: False
+#  local_update_steps: 1
+  total_round_num: 200
+  share_local_model: False
+  sample_client_num: 16
+data:
+  root: data/
+  type: fs_contest_data
```

Comment:
the data.type here should be graph-dt. Please check the correctness of the scripts.